### ISSUE TITLE
Does keybase.pub work on shared public folders?

### ISSUE THREAD

**Issue by puja108** [keybase/keybase-issues#2311]
Not sure if this works already, but couldn't find a way. We created a shared folder `/keybase/public/puja,othylmann/`, which works, but were wondering if there's a way to access that public folder on `keybase.pub`. Not sure how the comma would be translated. Maybe this is rather a feature request ;)

**Comment by cjb** [comment]
Nope, no support for shared public folders on keybase.pub yet. Sorry!

**Comment by malgorithms** [comment]
@puja108 - what were you looking to do in the public folder with multiple users? Curious about the use cases, as we think about making this available on the website.

**Comment by puja108** [comment]
@malgorithms we were thinking about things we could do with keybase, not very specific, more like a saturday night thought, but at some point we came up with some kind of use case that would involve working collaboratively on something like a blog or similar, which could be available to the public over keybase.pub without having to mount it to some server.

**Comment by yeswecan** [comment]
@malgorithms got the exact same idea today. I've invited my colleague to work on a public blog kind of thing together on keybase, but it turns out there's no way of doing it right now. Could be a great feature.

**Comment by ShayBox** [comment]
Is there any way to do this yet? I've spent all day trying to find some way to access shared public folders in the browser
